src/app/tests/TestICDManager.cpp

```diff
@@ -161,6 +180,177 @@ class TestICDManager
         NL_TEST_ASSERT(aSuite, ctx->mICDManager.mOperationalState == ICDManager::OperationalState::IdleMode);
     }
 
+    /**
+     * @brief Test verifies that the ICDManager starts its timers correctly based on if it will have any messages to send
+     *        when the IdleModeDuration expires
+     */
+    static void TestICDModeDurationsWith0ActiveModeDurationWithoutActiveSub(nlTestSuite * aSuite, void * aContext)
+    {
+        TestContext * ctx = static_cast<TestContext *>(aContext);
+        typedef ICDListener::ICDManagementEvents ICDMEvent;
+
+        // Set FeatureMap - Configures CIP, UAT and LITS to 1
+        ctx->mICDManager.SetTestFeatureMapValue(0x07);
+
+        // Set that there are not matching subscriptions
+        ctx->mSubManager.SetReturnValue(false);
+
+        // Set New durations for test case
+        uint32_t oldActiveModeDuration_ms = ICDConfigurationData::GetInstance().GetActiveModeDurationMs();
+        ICDConfigurationData::ICDConfigurationData::GetInstance().SetDurations(
```

Comment:
duplication of `ICDConfigurationData::`

I would also suggest getting the instance in a short named variable at the beginning of the test and use it instead of repeating the long `ICDConfigurationData::GetInstance()`

Reply:
done